databooks/data_models/notebook.py

```diff
@@ -247,9 +243,11 @@ def parse_file(cls, path: Path | str, **parse_kwargs: Any) -> JupyterNotebook:
             raise ValueError(
                 f"Value of `content_type` must be `json` (default), got `{content_arg}`"
             )
-        return super(JupyterNotebook, cls).parse_file(
-            path=path, content_type="json", **parse_kwargs
-        )
+
+        with open(path, "r+") as file:
+            raw_json = file.read()
+
+        return JupyterNotebook.model_validate_json(json_data=raw_json)
```

Comment:
Can we favor the [`Path.read_text()`](JupyterNotebook.model_validate_json(json_data=raw_json)v) instead?

So this can be replaced by `return JupyterNotebook.model_validate_json(json_data=path.read_text())`

Reply:
Good tip, more concise, will change 👍